content/docs/guide/map_submission/map_zoning.md

```diff
@@ -9,60 +9,420 @@ weight: 4
+1. Noclip to the start of the map
+    - **g** by default or `noclip` in console 
+1. Click **+ Main**
+    - This will automatically create the **main track**, **stage 1** and put you in the **zoning mode** 
+2. Select one corner of the starting platform
+    - The zoning tool will automatically detect vertices and snap to them ( blue indicators in the video below )
+    - Make sure you are always snapping to vertices when possible
+3. Select the opposite corner of the starting platform
+    - If the platform you're zoning is not rectangular see the [Freeform Mode](#freeform-mode) section
+4. Choose the height of your zone ( please follow the [zone height requirements](#zone-height))
+    - Make sure the **destination entity** is inside of your zone
+    - The entity will turn green when encompassed by the region
+    - The entity is usually above any starting platform ( grey box in the video below )
+5. If necessary, change the [Safe Height](#safe-height) of your zone
+
+{{<hint info>}}
+
+Zoning non-rectangular platforms: [Freeform Mode](#freeform-mode).  
+Zoning in mid-air: [Box Mode](#box-mode).
+
+{{</hint>}}
+
+{{<hint info>}}
+
+You can always [Edit Zones](#zone-editing) after creating them.
+
+{{</hint>}}
+
+{{<video src="/videos/map_zoning/create_start_zone.mp4">}}
+
+
+## Creating Stages and Checkpoints
+**Stages** and **Checkpoints** are used differently depending on the gamemode.  
+Please follow the guide appropriate for your map.  
+
+{{<hint info>}}
+
+Checkpoints should be placed where they can be hit consistently for better split comparisons.  
+Middle/End of a hallway is generally preferred  over placing them right at the start. 
+
+{{</hint>}}
+
+{{<hint info>}}
+
+When zoning **multiple routes** or **around a hole in geometry** you should be adding [Additional Regions](#multiple-regions) to a single zone.
+
+{{</hint>}}
+
+{{<hint warning>}}
+
+On all **Surf** maps, make sure to create [Allow Bhop](#allow-bhop) zones where necessary.
+
+{{</hint>}}
+
+- [Surf: Linear](/guide/map_submission/map_zoning/#surf-linear)
+- [Surf: Staged](/guide/map_submission/map_zoning/#surf-staged)
+- [Surf: Staged-Linear](/guide/map_submission/map_zoning/#surf-staged-linear)
+- [Rocket / Sticky Jump: No Courses](/guide/map_submission/map_zoning/#rocket--sticky-jump-no-courses)
+- [Rocket / Sticky Jump: With Courses](/guide/map_submission/map_zoning/#rocket--sticky-jump-with-courses)
+- [Other Gamemodes](/guide/map_submission/map_zoning/#other-gamemodes)
+
+### Surf: Linear
+Linear Surf maps should have **1 stage** and **checkpoints** placed throughout the map for **comparisons**.  
+**Example: surf_atonement zones**
+![Linear Surf Zones](/images/map_zoning/zoning_surf_linear.png)
+
+### Surf: Staged
+Staged Surf maps should have a seperate **stage zone** ( called **segment** in the zoning menu ) for every **stage**.  
+Those **stage zones** can have **checkpoints** for better comparisons ( usually only used for **really long** stages ).  
```

Comment:
This sounds like this is calling an entire stage/segment a "stage zone", but a zone is just a volume in one specific spot of a map. Not sure if that's what you meant, but I think calling a segment/subsegment a "zone" is actually something some people/communities do, so it would be good to clearly differentiate from that.

"Staged Surf maps should have a Segment for every stage. Segments for very long stages can have additional Checkpoints added within them for extra timer splits for comparing runs."